<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Janela_Principal</class>
 <widget class="QMainWindow" name="Janela_Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>676</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Janela_Principal</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_16">
          <property name="text">
           <string>Converter Complexo</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Forma Retangular</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7">
            <item>
             <widget class="QLineEdit" name="campo_texto_real_conversao"/>
            </item>
            <item>
             <widget class="QLabel" name="label_11">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="campo_texto_imag_conversao"/>
            </item>
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>j</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Forma Polar           </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9">
            <item>
             <widget class="QLineEdit" name="campo_texto_modulo_conversao"/>
            </item>
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;|Z|&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="campo_texto_angulo_conversao"/>
            </item>
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600;&quot;&gt;θ&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QPushButton" name="converter_botao">
            <property name="text">
             <string>Converta</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="zerar">
            <property name="text">
             <string>Zerar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="label_17">
          <property name="text">
           <string>Calcular</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Numero Complexo 1  </string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <widget class="QLineEdit" name="campo_texto_real_n1"/>
            </item>
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="campo_texto_imag_n1"/>
            </item>
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>j</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Numero Complexo 2  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_texto_real_n2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_texto_imag_n2"/>
          </item>
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>j</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Resultado Complexo  </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_texto_real_resultado"/>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>+</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="campo_texto_imag_resultado"/>
          </item>
          <item>
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>j</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QPushButton" name="Soma">
            <property name="text">
             <string>Somar</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Subtrair">
            <property name="text">
             <string>Subrair</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Dividir">
            <property name="text">
             <string>Dividir</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="Multiplicar">
            <property name="text">
             <string>Multiplicar</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Calculos de Elétrica</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Tensão        </string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLineEdit" name="Tensao_real"/>
              </item>
              <item>
               <widget class="QLabel" name="label_20">
                <property name="text">
                 <string>+</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="Tensao_imaginario"/>
              </item>
              <item>
               <widget class="QLabel" name="label_21">
                <property name="text">
                 <string>j</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>Corrente     </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Corrente_Real"/>
            </item>
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Corrente_Imaginario"/>
            </item>
            <item>
             <widget class="QLabel" name="label_24">
              <property name="text">
               <string>j</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Impedância</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Impedancia_Real"/>
            </item>
            <item>
             <widget class="QLabel" name="label_26">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Impedancia_imaginario"/>
            </item>
            <item>
             <widget class="QLabel" name="label_27">
              <property name="text">
               <string>j</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15">
            <item>
             <widget class="QLabel" name="label_44">
              <property name="text">
               <string>Potência     </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Potencia_Real"/>
            </item>
            <item>
             <widget class="QLabel" name="label_46">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="Potencia_imaginario"/>
            </item>
            <item>
             <widget class="QLabel" name="label_45">
              <property name="text">
               <string>j</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QPushButton" name="Calcular_Eletrica">
          <property name="text">
           <string>Calcular</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>676</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
